### PR TITLE
Feat: sessions added to startify page (#51, #72)

### PR DESCRIPTION
**Pull request by zhengrc19**
TLDR: Looks if [neovim-session-manager](https://github.com/Shatur/neovim-session-manager) is installed and if there are saved sessions. If both suffice, a new list of sessions is added on top of MRU CWD and MRU in the startify theme.

Checks for installation of [neovim-session-manager](https://github.com/Shatur/neovim-session-manager) in lua/alpha/themes/startify.lua:162.

Creates buttons (with optional icons) according to the session filenames lua/alpha/themes/startify.lua:164-202.

Changes the numerical shortcuts according to number of sessions found in lines lua/alpha/themes/startify.lua:229,243.

Adds sessions to `section` in lua/alpha/themes/startify.lua:212, and adds this `section.sessions` to `config.layout` in lua/alpha/themes/startify.lua:267.

The icons are turned off by default, configured in lua/alpha/themes/startify.lua:61.

P.S. This is my first PR ever so if there are any bad practices please point them out to me. Thanks!

### CODE REVIEW

**Comment by itamark-targa** [comment]
@zhengrc19  Thanks! very helpful, that's the 1 feature I was really missing moving from vim to nvim.
I have a question - is there a way to have sessions saved based on named-session, and not based on a directory? my sessions are just group of files related to a single task/project, so saving based on directory is less convenient.

**Comment by zhengrc19** [comment]
> @zhengrc19 Thanks! very helpful, that's the 1 feature I was really missing moving from vim to nvim. I have a question - is there a way to have sessions saved based on named-session, and not based on a directory? my sessions are just group of files related to a single task/project, so saving based on directory is less convenient.
> 
> == EDIT === - never mind, I found out that the original "startify" plugin for Vim works also for Neovim, so that solves all of my problems :)

Hey @itamark-targa thanks for reaching out! Yes in fact I was using `vim-startify` on Neovim for quite a while and it definitely has all these features implemented. I see if I can implement the session naming feature into `alpha-nvim` in the future. Do you know how `vim-startify` does this, does it even utilize `neovim-session-manager`?
